index.html

```diff
@@ -27,6 +27,139 @@
                 <li><a href="#"><img class="social-ico" src="res/twitter.svg" alt="twitter"></a></li>
                 <li><a href="#"><img class="social-ico" src="res/m.svg" alt="meta"></a></li>
             </ul>
-        </section>    
+        </section>
+        <section class="works-section">
+            <div class="my-recent-work">
+                <h1>My Recent Works</h1>
+                <hr class="recent-hr">
+            </div>
+            <div class="works">
+                <div class="project-1">
+                    <img class="project-1-img" src="res/img_Placeholder.png">
+                    <h3>Multi-Post Stories</h3>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                    <div class="siklls">
+                        <a>css</a>
+                        <a>html</a>
+                        <a>bootstrap</a>
+                        <a>Ruby</a>
+                    </div>
+                    <a class="project1-btn">See Project</a>
+                </div>
+                <div class="project-2">
+                    <div class="project-2-text">
+                        <h3>Profesional Art </br> Printing Data</h3>
+                        <p>A daily selection of privately </br> personalized reads; no accounts or </br> sign-ups required. has been the </br> industry's standard</p>
+                    </div>
+
+                    <div class="siklls skills2">
+                        <a>html</a>
+                        <a>bootstrap</a>
+                        <a>Ruby</a>
+                    </div>
+                    <button class="project-2-btn">See Project</button>
+                </div>
+                <div class="project-2">
+                    <div class="project-2-text">
+                        <h3>Profesional Art </br> Printing Data</h3>
+                        <p>A daily selection of privately </br> personalized reads; no accounts or </br> sign-ups required. has been the </br> industry's standard</p>
+                    </div>
+
+                    <div class="siklls skills2">
+                        <a>html</a>
+                        <a>bootstrap</a>
+                        <a>Ruby</a>
+                    </div>
+                    <button class="project-2-btn">See Project</button>
+                </div>
```

Comment:
- [ ] Kindly make sure  all the buttons (e.g., "See project" &&  "Gey My Resume")) follow the button styling: Active, Hover, and Pressed (see design) like on Figma. You check all the colors and fonts used on this image below from the figma template.
![Screenshot from 2022-09-24 12-41-27](https://user-images.githubusercontent.com/83042742/192093569-0da5ab8d-5855-4cc3-8944-69af62b992e2.png)